### PR TITLE
Method to determine if a point is inside a shape #17

### PR DESCRIPTION
**Pull request by alexeyleping**
https://github.com/1chooo/uml-editor/issues/17

look at the code in the request, I'm not sure I understood the task correctly, but I didn't wait for a comment

### CODE REVIEW

**Comment by 1chooo** [comment]
Please remove https://github.com/1chooo/uml-editor/pull/23/commits/34010279e4568b830ff7d86a2fa66a5a2555b2b4 and https://github.com/1chooo/uml-editor/pull/23/commits/cb70ce0e97a3bf9af2f7a5b20c713d3371f9d132 because they are not related to this PRs.

File: src/Utils/MODES.java
Comment (1chooo) [inline src/Utils/MODES.java:1]: We have remove `final` in #18

File: src/Shapes/Shape.java [hunk 1]
```diff
@@ -1,59 +1,74 @@
 package Shapes;
 
-import java.awt.Graphics;
-import java.awt.Point;
-import java.awt.Font;
+import java.awt.*;
```
Comment (1chooo) [inline src/Shapes/Shape.java:3]: Please keep the below

```java
import java.awt.Graphics;
import java.awt.Point;
import java.awt.Font;
```

**Review by 1chooo — CHANGES_REQUESTED** [review]

File: src/Shapes/Shape.java (hunk 1 above)
Comment (alexeyleping) [reply src/Shapes/Shape.java:3]: If you specify only 3 packages, then the polygon class is missing, after adding it, all packages in the import are automatically substituted, how do we do it?

Comment (1chooo) [reply src/Shapes/Shape.java:3]: you can use

```java
import java.awt.Polygon;
```

instead of 
```java
import java.awt.*;
```

Comment (alexeyleping) [reply src/Shapes/Shape.java:3]: https://github.com/1chooo/uml-editor/pull/27
new

**Pull request closed without merging**